### PR TITLE
LCH-6423: Using prophecy trait. Php unit 10 compatibility.

### PR DESCRIPTION
**Pull request by narendradesai**

### CODE REVIEW

File: tests/Command/Backups/AcquiaCloudBackupCreateTest.php [hunk 1]
```diff
@@ -18,6 +19,8 @@
  */
 class AcquiaCloudBackupCreateTest extends AcquiaCloudBackupTestBase {
 
+  use ProphecyTrait;
```
Comment (amangrover90) [inline tests/Command/Backups/AcquiaCloudBackupCreateTest.php:22]: Better to add this trait in base test class.

**Review by amangrover90 — CHANGES_REQUESTED** [review]
Left a comment, better to add this trait into another common trait or base classes.

File: tests/Command/ContentHubHealthCheckWebhookStatusTest.php [hunk 2]
```diff
@@ -26,6 +27,8 @@
  */
 class ContentHubHealthCheckWebhookStatusTest extends ContentHubCommandTestBase {
 
+  use ProphecyTrait;
+
```
Comment (amangrover90) [inline tests/Command/ContentHubHealthCheckWebhookStatusTest.php:31]: Better to add this in ContentHubCommandTestBase than adding in all the inheritors.

**Review by amangrover90 — CHANGES_REQUESTED** [review]
Left a comment.

**Review by saxumVermes — APPROVED** [review]

**Pull request merged**